core/variant/native_ptr.h

```diff
@@ -34,6 +34,7 @@
 #include "core/math/audio_frame.h"
 #include "core/variant/method_ptrcall.h"
 #include "core/variant/type_info.h"
+#include "servers/text_server.h"
```

Comment:
We were discussing with @reduz on chat that `core` should likely not depend on `servers`, so these structs might need to be moved to a `core` header. Not sure where exactly though.

Reply:
Or simply moving `GDVIRTUAL_NATIVE_PTR` calls to the text server header should do the same.

Reply:
Ah yes, perfect.